template.yaml

```diff
@@ -182,3 +182,64 @@ Resources:
       KmsKeyId: '{{resolve:ssm:LogsKmsKeyArn}}'
       LogGroupName: !Sub '/aws/lambda/${AWS::StackName}-download-warning-page'
       RetentionInDays: 30
+
+  SendEmailRequestToNotifyFunction:
+    #checkov:skip=CKV_AWS_115:Defined in Globals section
+    #checkov:skip=CKV_AWS_116:Unsure of what will call the function currently - need revision at a later point
```

Comment:
Maybe update this comment to say that the queue itself will handle any dead-lettering